### ISSUE TITLE
File "not included into executable" with 5.2.1

### ISSUE THREAD

**Issue by allie47** [vercel/pkg#1202]
**Environment:** Linux (Debian, x86_64)
I use usocket.node, and prep the app via websocket prior to processing it via pkg.

**When using:** 
Node v14.17.0
pkg@5.2.1 or 5.2.0

I can run the pkg without issue, no error, it creates the files and nothing seems odd. Up until I try to run the resulting app - where I immediately get following error:
```
pkg/prelude/bootstrap.js:1614
      throw error;
      ^

Error: File '/**/Backend/node_modules/usocket/build' was not included into executable at compilation stage. Please recompile adding it as asset or script.
    at error_ENOENT (pkg/prelude/bootstrap.js:539:17)
    at readFileFromSnapshot (pkg/prelude/bootstrap.js:891:29)
    at Object.readFileSync (pkg/prelude/bootstrap.js:938:18)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1095:18)
    at Module.load (internal/modules/cjs/loader.js:933:32)
    at Function.Module._load (internal/modules/cjs/loader.js:774:14)
    at Module.require (internal/modules/cjs/loader.js:957:19)
    at Module.require (pkg/prelude/bootstrap.js:1593:31)
    at require (internal/modules/cjs/helpers.js:88:18)
    at bindings (/snapshot/Backend/node_modules/bindings/bindings.js:112:48)
    at Object.<anonymous> (/snapshot/Backend/node_modules/usocket/index.js:6:32)
    at Module._compile (pkg/prelude/bootstrap.js:1686:22)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)
    at Module.load (internal/modules/cjs/loader.js:933:32)
    at Function.Module._load (internal/modules/cjs/loader.js:774:14)
    at Module.require (internal/modules/cjs/loader.js:957:19) {
  errno: -2,
  code: 'ENOENT',
  path: '/snapshot/Backend/node_modules/usocket/build/',
  pkg: true
}
```

**However when using:**
Node v14.16.1
pkg@5.1.0

It starts alright, no errors whatsoever, everything works as expected.

All other dependencies and settings are exactly the same in both cases.

If there is any further info I can try to provide, please let me know.

**Comment by jesec** [comment]
Probably resolved by 52ddf231ded0b7a17562c95a8697811c82a6c1e9 .

**Comment by erossignon** [comment]
duplicate #1191, this is happening with "bindings" and 5.2.
